pkg/arvo/tests/sys/zuse/crypto/ed25519.hoon

```diff
@@ -0,0 +1,156 @@
+::  tests for the ed25519 signature algorithm
+::
+/+  *test
+=,  ed:crypto
+::
+|%
+::
+::  Test vectors from Section 7.1 of RFC 8032:
+::  https://tools.ietf.org/html/rfc8032#section-7.1
+::
++$  vector  [seed=@ux pk=@ux msg=@ux sig=@ux]
+::
+++  test-vectors
+  ^-  tang
+  |^  ;:  weld
+        %+  category  "puck"
+        (zing (turn vectors check-puck))
+        %+  category  "sign"
+        (zing (turn vectors check-sign))
+        %+  category  "veri"
+        (zing (turn vectors check-veri))
+      ==
+  ::
+  ++  check-puck
+    |=  vector
+    %+  expect-eq
+      !>  pk
+      !>  `@ux`(puck seed)
+  ::
+  ++  check-sign
+    |=  vector
+    %+  expect-eq
+      !>  sig
+      !>  `@ux`(sign msg seed)
+  ::
+  ++  check-veri
+    |=  vector
+    %+  expect-eq
+      !>  %.y
+      !>  (veri sig msg pk)
```

Comment:
Is there a shorthand for `expect-eq %.y`? If not, might be convenient to have a bare `expect` arm.

Reply:
Definitely! Feel free to add it to `lib/test`.